tw-tasks-core/src/main/java/com/transferwise/tasks/utils/ClientIdUtils.java

```diff
@@ -2,11 +2,9 @@
 
 import com.transferwise.common.baseutils.ExceptionUtils;
 import java.net.InetAddress;
-import lombok.experimental.UtilityClass;
 import org.apache.commons.lang3.StringUtils;
 
-@UtilityClass
-public class ClientIdUtils {
+public abstract class ClientIdUtils {
```

Comment:
Let's use private constructor for it instead, as Joshua Bloch suggest in Effective Java. The common pattern is 

```java
public final class Utility { 
  private Utility() {
    throw new AssertionError();
  }
}
```